roles/openshift_storage_glusterfs/tasks/glusterfs_deploy.yml

```diff
@@ -0,0 +1,108 @@
+---
+- fail:
+    msg: Only one GlusterFS nodeselector key pair should be provided
+  when: "openshift_storage_glusterfs_nodeselector.keys() | count > 1"
+
+- fail:
+    msg: There must be at least three GlusterFS nodes specified
+  when: "groups.oo_glusterfs_to_config | count < 3"
+
+- name: Delete pre-existing GlusterFS resources
```

Comment:
By default they are not run, as `openshift_storage_glusterfs_wipe` is False. However, if something went wrong during the deployment of GlusterFS (e.g. you specified the wrong nodes or the wrong storage devices) there's no easy way to clean that up without recreating the resources.

Further, if something goes wrong while deploying heketi AFTER it has already formatted the storage devices, subsequent attempts to deploy heketi will fail unless the storage devices are wiped clean.